### ISSUE TITLE
gradient accumulation - dealing with large datasets

### ISSUE THREAD

**Issue by qualiaMachine** [UW-Madison-DataScience/ML-X-Nexus#94]
This effectively emulates the behavior of training with a larger batch size without requiring all the data to be processed at once in memory.

Can probably place this under Toolbox/Compute unless someone has a better idea for a new category that could capture topics like this.
